lib/start_game.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './start_game_vars'
+module StartGame
+  def win_methods
+    player_input
+    invalid
+    avaliable_inputs
+    board_interface
+  end
+
+  include StartGameVar
+
+  def start_game
+    game_numbers
+    until winer
+      win_methods
+      break if winer
+
+      if @avaliable.empty?
+        p 'draw'
+        break
```

Comment:
You should not use any function that prints to the console outside of `main.rb`. That includes `p`.